### ISSUE TITLE
Go 1.12.1 Trojan:Win32/Skeeyah.I detected

### ISSUE THREAD

**Issue by molxyz** [golang/go#30969]
I'm trying to install Go version 1.12.1 from the official site https://golang.org/doc/install?download=go1.12.1.windows-amd64.msi on my Windows 10. 
But Windows keeps saying this file has `Trojan:Win32/Skeeyah.I` which I can't find any info for this virus on the internet. Here's some info:
```
Details: This program is dangerous and executes commands from an attacker.

Affected items:

containerfile: C:\Users\username\Downloads\go1.12.1.windows-amd64.msi

file: C:\Users\username\Downloads\go1.12.1.windows-amd64.msi->go.cab->fil007491FAD0C78A02597DB0165DE0E2D5

webfile: C:\Users\username\Downloads\go1.12.1.windows-amd64.msi|https://dl.google.com/go/go1.12.1.windows-amd64.msi|pid:10628,ProcessStart:131975807114512553
```
Does anyone know if this is a false positive?

**Comment by Algoru** [comment]
Same here. The SHA256 sum doesn't not correspond.

SHA256 at Golang site: 034b5bbe2337f69c5f9c0b533171b1a7a32ad30ca8dff963105aaf026601150a
SHA256: 13515401fc5e8e2959ee37413f9d2af934e2bfbba87a2a4d7a1c6058a9dbe6d5

**Comment by diamondburned** [comment]
Can't confirm.

```sh
hanaharu ~/ sha256sum go1.12.1.windows-amd64.msi 
034b5bbe2337f69c5f9c0b533171b1a7a32ad30ca8dff963105aaf026601150a  go1.12.1.windows-amd64.msi
```

**Comment by bradfitz** [comment]
You might have an existing virus or malware that's MITMing your downloads.

See https://golang.org/doc/faq#virus

**Comment by Algoru** [comment]
If I use Firefox instead of Chrome the save file is downloaded... This is normal?

**Comment by bradfitz** [comment]
Maybe your machine is infected with software that targets Chrome.

In any case, your machine's probably infected. You might want to clean it or nuke it and reinstall.

**Comment by diamondburned** [comment]
`wget` it :)

**Comment by RobertPaasche** [comment]
Same problem here. Downloading 1.11.6 works fine.
Version 1.12.1 triggers chrome thread protection and downloading any other way the anti-virus is triggered.

Tested on a completly fresh system.

Downloading the msi with an linux machine triggers anti-virus too. So there seems to be a Problem with the msi on the server.

**Comment by muzykantov** [comment]
Downloaded Zip hash is correct, but my AV makes alarm. Virustotal shows 2 trojans

https://www.virustotal.com/gui/file/644d6d5ecbdbfe13a98851dcb51cba502735d8bdabc2dc646ca96bb22db40230/detection

Downloading 1.11.6 works fine.

**Comment by molxyz** [comment]
I think this case is false positive. Got the windows defender to allow so I could get the download. Then I ran the SHA256 hash check and it matched with the hash on golang site. Then I uploaded the download to virustotal.com, the analysis showed it passed with all flying green color. 

Still not sure so I downloaded it again on commandline `curl -O https://storage.googleapis.com/golang/go1.12.1.windows-amd64.msi` , then I checked the sha256 hash again, it matched. Ran it thru virustotal.com again, all passed in green.

Also not sure so I scanned the whole system, took almost all day... No virus found.
